src/modules/flow/iio/nodes.c

```diff
@@ -943,6 +943,154 @@ light_open(struct sol_flow_node *node, void *data, const struct sol_flow_node_op
 }
 
 static bool
+intensity_both_create_channels(struct iio_double_data *mdata, int device_id)
+{
+    mdata->iio_base.device = sol_iio_open(device_id, &mdata->iio_base.config);
+    SOL_NULL_CHECK(mdata->iio_base.device, false);
+
+    mdata->channel_val = iio_add_channel(mdata->scale, mdata->offset, "in_intensity_both", &mdata->iio_base);
```

Comment:
@fulong82 Thanks for your test. I am using the TSL2563 as below. Please let me know which the sensor name you are testing, regarding other sensors RPR0521/LTR501/APDS9300/HID ALS, do you know if they are using the in_intensity_offset or in_intensity_both_offset ?
 cat name
tsl2563
root@intel-quark:/sys/bus/iio/devices/iio:device1# ls -l
-r--r--r--    1 root     root          4096 Sep 27 15:27 dev
drwxr-xr-x    2 root     root             0 Sep 27 15:27 events
-rw-r--r--    1 root     root          4096 Sep 27 15:27 in_illuminance0_input
-rw-r--r--    1 root     root          4096 Sep 27 15:27 in_intensity_both_calibscale
-rw-r--r--    1 root     root          4096 Sep 27 15:27 in_intensity_both_raw
-rw-r--r--    1 root     root          4096 Sep 27 15:27 in_intensity_ir_calibscale
-rw-r--r--    1 root     root          4096 Sep 27 15:27 in_intensity_ir_raw
-r--r--r--    1 root     root          4096 Sep 27 15:27 name
drwxr-xr-x    2 root     root             0 Sep 27 15:27 power
lrwxrwxrwx    1 root     root             0 Sep 27 15:27 subsystem -> ../../../../../../../bus/iio
-rw-r--r--    1 root     root          4096 Sep 27 15:27 uevent